Update whats-new.md (#2328)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeOnline/whats-new.md

For now, it is possible to switch back to the existing EAC (often called the "classic" EAC), but at a future date, the classic EAC will be retired.

> ## September 2020 (Ignite Release) 

Reviewer: ```suggestion
## September 2020 (Ignite release) 
```

Here are some of the changes and new features we introduced in the modern EAC in September 2020. 

> ### Groups – Creation and Management 

Reviewer: ```suggestion
### Groups – creation and management 
```

> The admins can now create Groups from the modern EAC. Currently, they can create and manage M365 groups, Distribution groups and Mail-enabled Security groups.

Reviewer: ```suggestion
The admins can now create Groups from the modern EAC. Currently, they can create and manage Microsoft 365 groups, Distribution groups, and Mail-enabled Security groups.
```

> 3 types of Groups that the admins can now create and manage are:  

Reviewer: ```suggestion
Three types of groups that the admins can now create and manage are:  
```

> **M365 Groups** – They are the recommended Group type and they allow for effective collaboration by providing them shared email and a shared workplace. 

Reviewer: ```suggestion
**Microsoft 365 groups** – They are the recommended group type and they allow for effective collaboration by providing shared email and a shared workplace. 
```

> **Distribution Groups** – They are the most used Group type and they allow you to send email to all members of the list. 

Reviewer: ```suggestion
**Distribution groups** – They are the most used group type and they allow you to send email to all members on the list. 
```

> **Mail-enabled Security Groups** – They give the members access to various resources like OneDrive, SharePoint and admin roles. 

Reviewer: ```suggestion
**Mail-enabled security groups** – they give the members access to various resources like OneDrive, SharePoint, and admin roles. 
```

> ### Normal/Compact list view in Groups 

Reviewer: ```suggestion
### Normal/Compact list view in groups 
```

> The admins can now choose to view the list of groups in normal or compact list view way. Compact list option allows the admins to view more entries per page. 

Reviewer: ```suggestion
Admins can now choose to view the list of groups in normal or compact list view. The compact list option allows admins to view more entries per page. 
```

> ### Naming Policy for Groups 

Reviewer: ```suggestion
### Naming Policy for groups 
```

> The admins now have command over the Group naming policy. They can now add prefixes and suffixes for the future groups that will be created. They can also block specific words from being used in group names and aliases. 

Reviewer: ```suggestion
Admins now have command over the group naming policy. They can now add prefixes and suffixes for the future groups that will be created. They can also block specific words from being used in group names and aliases. 
```

> ### Upgrade Distribution Group 

Reviewer: ```suggestion
### Upgrade distribution group 
```

> The admins can now upgrade the distribution groups into the recommended Microsoft 365 (previously O 365 groups) with few clicks. 

Reviewer: ```suggestion
Admins can now upgrade the distribution groups into the recommended Microsoft 365 groups (previously Office 365 groups) with a few clicks. 
```

### Opt-in/out 

> The users now have a single click Opt-in toggle button available in classic and the new Exchange Admin Center. The button allows for easy navigation between two portals and now users can try and access the new portal seamlessly. 

Reviewer: ```suggestion
The users now have a single click **Opt-in** toggle button available in the classic and the new Exchange admin center. The button allows for easy navigation between two portals and now users can access the new portal seamlessly. 
```

> ### Support Assistant 

Reviewer: ```suggestion
### Support assistant 
```

The admins can now get their queries resolved without going anywhere else. The support assistant feature allows users to resolve the queries without leaving the EAC window. In case the user is not satisfied with the provided solution, the user can even raise a ticket and register their issues. 

> ### Educational Navigation Video 

Reviewer: ```suggestion
### Educational navigation video 
```

> An educational navigation video is now available on the very first tile ‘Training and Guide’ in the new EAC portal. The video allows the users to understand the basic navigation of the new portal. You can also access the video here: https://www.microsoft.com/en-us/videoplayer/embed/RE4FqDa. 

Reviewer: ```suggestion
An educational navigation video is now available on the very first tile "Training and Guide" in the new EAC portal. The video allows the users to understand the basic navigation of the new portal. You can also access the video [here](https://www.microsoft.com/videoplayer/embed/RE4FqDa.
```

> ### Recipients – MS Documents Updated 

Reviewer: ```suggestion
### Recipients – Microsoft documents updated 
```

> Going forward with helping the users with their queries related to new Exchange Admin Center portal, the Microsoft documentation is also complete for the high traffic pages under Recipients.

Reviewer: ```suggestion
In moving forward to help the users with their queries related to the new Exchange admin center portal, we have also completed the Microsoft documentation for high traffic pages under "Recipients".
```
